src/main/io/gps.h

```diff
@@ -104,6 +104,9 @@ typedef enum {
     GPS_MESSAGE_STATE_INIT,
     GPS_MESSAGE_STATE_SBAS,
     GPS_MESSAGE_STATE_GALILEO,
+    GPS_MESSAGE_STATE_INITIALIZED,
+    GPS_MESSAGE_STATE_SETTING_AIRBORNE,
+    GPS_MESSAGE_STATE_SETTING_PEDESTRIAN,
```

Comment:
Maybe GPS_MESSAGE_STATE_SETTING_AIRBORNE (or _ARMED) / GPS_MESSAGE_STATE_SETTING_DISARMED

Reply:
Maybe I can remove "SETTING" to be more inline with the naming of the other states, but I don't think ARMED/DISARMED has anything to do in this particular enum. ie: maybe in the future other commands are sent when arming/disarming.